build_library/sysext_prod_builder

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+# Copyright (c) 2023 by the Flatcar Maintainers.
+# Use of this source code is governed by the Apache 2.0 license.
+
+# Helper script for building OS images w/ sysexts included.
+# Called by build_image -> prod_image_util.sh.
+# This is a separate script mainly so we can trap EXIT and clean up our mounts
+#  without interfering with traps set by build_image.
+
+# We're in build_library/, script root is one up
+SCRIPT_ROOT="$(cd "$(dirname "$(readlink -f "$0")")/../"; pwd)"
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+switch_to_strict_mode
+
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+
+# Create a sysext from a package and install it to the OS image.
+# Conventions:
+# - For each <group>/<package>, <group>_<package>_pkginfo will be built. Can be used in subsequent calls
+#   to build dependent sysexts.
+# - If ${BUILD_LIBRARY_DIR}/sysext_mangle_<group>_<package> exists it will be used as FS mangle script
+#   when building the sysext.
+create_prod_sysext() {
+  local BOARD="$1"
+  local output_dir="$2"
+  local workdir="$3"
+  local base_sysext="$4"
+  local install_root="$5"
+  local name="$6"
+  local grp_pkg="$7"
+  local pkginfo="${8:-}"
+
+  local pkginfo_opt=""
+  local manglefs_opt=""
+
+  local msg="Installing ${grp_pkg}' in sysext ${name}.raw"
+
+  # Include previous sysexts' pkginfo if supplied
+  if [[ -n "${pkginfo}" ]] ; then
+    if [[ ! -f "${output_dir}/${pkginfo}" ]] ; then
+      die "Sysext build '${grp_pkg}': unable to find package info at '${output_dir}/${pkginfo}'."
+    fi
+    msg="${msg} w/ package info '${pkginfo}'"
+    pkginfo_opt="--base_pkginfo=${output_dir}/${pkginfo}"
+  fi
+
+  # Include FS mangle script if present
+  if [[ -x "${BUILD_LIBRARY_DIR}/sysext_mangle_${name}" ]] ; then
+    manglefs_opt="--manglefs_script=${BUILD_LIBRARY_DIR}/sysext_mangle_${name}"
+    msg="${msg}, FS mangle script 'sysext_mangle_${name}'"
+  fi
+
+  info "${msg}."
+  
+  sudo "${SCRIPTS_DIR}/build_sysext" \
+            --board="${BOARD}" \
+            --image_builddir="${workdir}/sysext-build" \
+            --squashfs_base="${base_sysext}" \
+            --generate_pkginfo \
+            ${manglefs_opt} ${pkginfo_opt} \
+            "${name}" "${grp_pkg}"
+
+  sudo mv "${workdir}/sysext-build/${name}.raw" "${workdir}/sysext-build/${name}_pkginfo.raw" \
+                              "${workdir}/sysext-build/${name}"_*.txt "${output_dir}"
+
+  sudo mkdir -p "${install_root}"/usr/share/flatcar/sysext
+  sudo install -m 0644 -D "${output_dir}/${name}.raw" "${install_root}"/usr/share/flatcar/sysext/
+
+  sudo mkdir -p "${install_root}"/etc/extensions/
+  sudo ln -sf "/usr/share/flatcar/sysext/${name}.raw" "${install_root}/etc/extensions/${name}.raw"
```

Comment:
I believe I just used that because that's how we ship `/usr/share/flatcar/etc/` and other Flatcar defaults. This decision was never discussed widely - happy to have that discussion. It's very easy to change this because this file is the only place where that path is generated, filled, and used (for the softlink).

Reply:
Can we make it `/usr/lib/flatcar/sysext` then?